### DISCUSSION TITLE
Subflow - Need to position the sub nodes inside a node

### DISCUSSION THREAD

**Discussion by FarrukhXeb** [xyflow/xyflow#3125] [Q&A]
### Issue
I'm struggling to position the sub-nodes inside a node horizontally. A parent node has specific width of 1500px and the nodes are traversed inside the parent node dynamically by uploading JSON data. I want the sub-nodes which reach the end of the parent nodes' width to be then aligned to the next row and traverse forward. Also, the height of the Parent node is to be increased.
Here is an example
![image](https://github.com/wbkd/react-flow/assets/43153347/fc9dd1f1-c6c2-4d02-9586-e90a5ea66198)

I know it's going to be related to some calculations of the nodes' x and y coordinates but that's confusing. Help would be much obliged.

**Comment by mayurjadhav2002** [comment]
same doubt here
